ivy_models/bert/layers.py

```diff
@@ -0,0 +1,332 @@
+import ivy
+import math
+
+
+class Embedding(ivy.Module):
```

Comment:
I think we have an embedding class in ivy itself, are you able to unify the behaviours here into the existing class in ivy?

Reply:
sure, I'll keep it here for now, but once this is updated in ivy, please open a PR for removing the class here and refactor it to the one from `ivy.Embedding`, thanks!